http/http_test.go

```diff
@@ -72,7 +86,7 @@ func testHttpData(t *testing.T, method string, token string, addr string, body i
 	}
 
 	resp, err := client.Do(req)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "checkRedirect disabled for test") {
```

Comment:
Not a huge deal, but instead of checking the contents of the string you could save the error as a variable and check equivalence.

```
var errCheckRedirectDisabled error = errors.New("checkRedirect....")
```

Then later do:
```
if err != nil && err != errCheckRedirectDisabled
```